client/my-sites/sidebar/index.jsx

```diff
@@ -493,6 +493,30 @@ export class MySitesSidebar extends Component {
 		);
 	}
 
+	trackHostingClick = () => {
+		this.trackMenuItemClick( 'hosting' );
+		this.onNavigate();
+	};
+
+	hosting() {
+		const { translate, path, site, isAtomicSite, siteSuffix } = this.props;
+
+		if ( ! site || ! isAtomicSite ) {
+			return null;
+		}
+
+		return (
+			<SidebarItem
+				label={ translate( 'Hosting' ) }
+				selected={ itemLinkMatches( '/hosting', path ) }
+				link={ `/hosting${ siteSuffix }` }
+				onNavigate={ this.trackHostingClick }
+				preloadSectionName="hosting"
```

Comment:
✅ Verified that hovering over the "hosting" link loads the hosting bundle:

<img width="956" alt="Screen Shot 2019-10-07 at 10 48 04 AM" src="https://user-images.githubusercontent.com/1270189/66335516-04183080-e8f0-11e9-8124-ee3e73b6bd2c.png">